src/test/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck " + getName() + " started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck " + getName() + " finished my work.");
```

Comment:
The stop message should be informative and consistent with the start message. Consider rephrasing the stop message to match the format of the start message. For example: 'Truck <name> has stopped working.'